Minor changes (#42)
OpenPodcastAPI/api-specs: docs/subscriptions/index.md

}
:::

:::{admonition} Tombstoning
:class: note

> As is implicit from the description of the servers-side behavior for the different calls, servers SHOULD hold all previous `guid` and `feed_url` field data with a link to the succeeding data (such that a path of values can be followed) or with a link to the most recent data. This in order to handle situations where clients submit old data. For example:

Reviewer: I think it could be good to point out that only the old GUID and the link to the new entry is necessary to store and that the rest can be deleted.

We only describe this behaviour here:
>If the GUID isn’t already present, create a new subscription entry and update the existing entry’s new_guid field to point to the newly created entry

It could also be enough to just add a sentence to this section like: `The fields GUID and NEW_GUID of the existing entry SHOULD be kept indefinitely for referencing.`

> * A user finds a podcast, whose URL had changed, and adds the old URL in the app. Because the client does not have that URL in its database, it recognizes the podcast as 'new' and POSTs this old (not-current) `feed_url` to the /subscriptions endpoind. If the user is already subscribed to the podcast (with the current feed URL) this would lead to a duplicate subscription.

Reviewer 2: ```suggestion
* A user finds a podcast, whose URL had changed, and adds the old URL in the app. Because the client doesn't have the old URL in its database, it recognizes the podcast as **new** and POSTs the `feed_url` to the `/subscriptions` endpoint. If the user is already subscribed to the podcast (with the current feed URL) this would lead to a duplicate subscription.
```

> * A user has a device that they didn't use for a very long time. In that time, a podcaster added a GUID in their feed, leading to updated data in this field. When the client connects to the server again to pull all episode changes since the last connect and retrieves episodes with their current subscription `guid`, which it won't recognise the subscription and fail the episode status updates.

Reviewer 2: ```suggestion
* A user has a device that they didn't use for a very long time. In that time, a podcaster added a GUID in their feed, leading to updated data in this field. When the client connects to the server again to pull all episode changes since the last connection, it retrieves episodes with their current subscription `guid`. The client won't recognize the subscription and fail to update the status of episodes.
```
